flipperdevices/flipperzero-firmware
# Add MacOS and Linux set up guide

# What's new

- Add set up guide for MacOS
- Add set up guide for Linux
- Remove -j flag from instruction (not yet needed)

# Verification 

- Find clear Mac without any library
- Follow guide
- Check that all command execute correctly

- Find clear Debian/Ubuntu without any library
- Follow guide
- Check that all command execute correctly

# Checklist (do not modify)
